notify/nfstatus/integration.go

```diff
@@ -93,6 +98,10 @@ func (n *statusCaptureNotifier) Notify(ctx context.Context, alerts ...*types.Ale
 	retry, err := n.upstream.Notify(ctx, alerts...)
 	duration := time.Since(start)
 
+	if n.notificationHistorian != nil {
+		n.notificationHistorian.Record(ctx, alerts, err)
```

Comment:
Let's add more information here: Attempt # and duration. Perhaps retry status would be useful too.

Reply:
valid points 👍 I'll address it when moving the notification historian to `grafana/alerting`